cypress/e2e/signIn.cy.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+describe('Sign in:', () => {
+  beforeEach(() =>{
+    cy.visit('https://the-internet.herokuapp.com/login')
+  })
+
+  it('Sign in with valid creds', () => {
+    cy.login('tomsmith', 'SuperSecretPassword!')
```

Comment:
Don't forget to put ; 
```suggestion
    cy.login('tomsmith', 'SuperSecretPassword!');
```